taxi_service/settings.py

```diff
@@ -24,7 +24,7 @@
     "django-insecure-8ovil3xu6=eaoqd#-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
 )
 
-# SECURITY WARNING: don"t run with debug turned on in production!
+# SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
 ALLOWED_HOSTS = []
```

Comment:
The `ALLOWED_HOSTS` list is empty. In production, you should specify the domain names or IP addresses your Django site can serve.